pkg/Library/Mediapipe/Pose.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+({
+shouldUpdate({image}) {
+  return image;
+},
+async update({image}, state, {service}) {
+  let pose = await service({kind: 'PoseService', msg: 'classify', data: {image}});
+  if (!pose) {
+    pose = {
+      poseLandmarks: {
+        11: [0, 10, 0], // shoulder a
```

Comment:
sorry, i'm confused, how did you choose these defaults (and why only these four)?

Reply:
The 'is hand raise' test I built uses these four points (shoulders and elbows) to detect raising of hand.